task_01.py

```diff
@@ -0,0 +1,78 @@
+#!user/bin/env python
+# -*- coding: utf-8 -*-
+"""Module that converts degrees in either fahrenheit or celsius into
+   either celsius or kelvin"""
+
+
+import decimal
+
+
+ABSOLUTE_DIFFERENCE = decimal.Decimal('273.15')
+
+
+def fahrenheit_to_celsius(degrees):
+    """Does math to return conversion from f to c as a decimal.
+
+    Args:
+        degrees (dec): degree to be converted.
+
+    Returns:
+        (dec) converted to celsius.
+
+    Examples:
+        fahrenheit_to_celsius(53)
+        >>>> '11.66666666666666666666666667'
+    """
+    deg_far = decimal.Decimal(degrees)
+    val_a = decimal.Decimal('32')
+    val_b = decimal.Decimal('5')
+    val_c = decimal.Decimal('9')
```

Comment:
It's unnecessary to keep redeclaring everything as a decimal -- the reason is that any interactions between an integer and a decimal return a decimal automatically (as the more complex type) to the conversion is implicit.